cmd/ipfs/daemon.go

```diff
@@ -421,6 +422,7 @@ func daemonFunc(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment
 
 	// initialize metrics collector
 	prometheus.MustRegister(&corehttp.IpfsNodeCollector{Node: node})
+	prometheus.MustRegister(promversion.NewCollector("go_ipfs"))
```

Comment:
Double checking: does this mean we _have_ to pass those build flags? dist.ipfs.io doesn't build using the makefile.